website/src/components/Changelog/Headless.tsx

```diff
@@ -16,6 +16,13 @@ export default function Headless() {
         </ul>
       </Entry>
       */}
+      <Entry version="1.1.6" date={new Date("2024-08-09")}>
```

Comment:
I think you need this in the GUI one too

Reply:
For the GUI, I already added it in the other PR :)